apps/web/components/OrderSummary/OrderSummary.vue

```diff
@@ -34,6 +34,10 @@ import { cartGetters } from '@plentymarkets/plentymarkets-sdk/packages/sdk/src';
 import type { OrderSummaryPropsType } from '~/components/OrderSummary/types';
 
 const props = defineProps<OrderSummaryPropsType>();
+const totals = cartGetters.getTotals(props.cart).total;
+const subTotal = cartGetters.getTotals(props.cart).subtotal;
+const shippingPrice = cartGetters.getShippingPrice(props.cart);
+const vatAmmount = Number(cartGetters.getTotals(props.cart)?.vatAmount);
```

Comment:
```suggestion
const totals = cartGetters.getTotals(props.cart);
const total = totals.total;
const subTotal = totals.subtotal;
const vatAmmount = Number(totals?.vatAmount);

const shippingPrice = cartGetters.getShippingPrice(props.cart);
```
**suggestion**: Reduce number of times we query the getter.

If accepted, this also requires a change from `totals` to `total` in line 24.